shell/src/chain_manager.rs

```diff
@@ -631,6 +633,43 @@ impl ChainManager {
                             }.into(),
                             topic: ShellChannelTopic::ShellEvents.into(),
                         }, Some(ctx.myself().into()));
+
+                    // handle operations
+                    // Note: When injecting the first block with the protocol activation, there are no validation passes (therefore no operations)
+                    // so we handle operations only above level 1 
+                    if level > 1 {
+                        // safe unwrap after above first block
+                        let operations = inject_data.operations.clone().unwrap();
+                        let header = block_header_with_hash.header;
+                        let op_paths = inject_data.operation_paths.clone().unwrap();
+
+                        // iterate through all validation passes 
+                        for (idx, ops) in operations.iter().enumerate() {
+                            let opb = OperationsForBlock::new(header.message_hash().unwrap(), idx as i8);
```

Comment:
@adonagy 
check all `message_hash().unwrap()` in this pr,
most of them can by written as `message_hash().?`